### DISCUSSION TITLE
Drop to floor [Game Studio]

### DISCUSSION THREAD

**Discussion by evargas-art** [stride3d/stride#1648] [Ideas]
Hi guys, I believe it would save a lot of time having some way (shortcut, menu, icon?) to quickly drop an object on the ground right after placing it on the scene (ground meaning the nearest surface downwards).
I know it can be done with physics component, but that requires an extra step in what is a relatively frequent task.

**Comment by NicusorN5** [comment]
I'd actually give a try implementing this in the Game Studio, thanks for the idea!

I will create a PR when finished.
